packages/airnode-utilities/src/logging.ts

```diff
@@ -0,0 +1,19 @@
+// @ts-ignore
+// eslint-disable-next-line no-console
+const callConsoleFn = (functionName: string, arg: any) => console[functionName](arg);
+
+export const createLogMessage = (args: any[]) =>
+  args
+    .map((arg) => {
+      if (arg instanceof Object) {
+        return JSON.stringify(arg, null, 2);
+      }
+
+      return arg.toString();
+    })
+    .join(' ');
+
+export const log = (...args: any[]) => callConsoleFn('log', createLogMessage(args));
+export const logWarn = (...args: any[]) => callConsoleFn('warn', createLogMessage(args));
+export const logError = (...args: any[]) => callConsoleFn('error', createLogMessage(args));
+export const logTrace = (...args: any[]) => callConsoleFn('trace', createLogMessage(args));
```

Comment:
TBH I don't like this approach much. I would rather have something more standard, e.g.
```js
import * as log from './logging';

log.info("Something to log")
```

Reply:
I'd also prefer something like https://github.com/api3dao/airnode/blob/master/packages/airnode-examples/src/cli.ts#L34 (but I am biased since I wrote that part)